tests/v2/validation/clustertemplates/provisioning/provisioning_clustertemplate_test.go

```diff
@@ -0,0 +1,295 @@
+//go:build (validation || sanity) && !infra.any && !infra.aks && !infra.eks && !infra.rke2k3s && !infra.gke && !infra.rke1 && !cluster.any && !cluster.custom && !cluster.nodedriver && !extended && !stress
+
+package clustertemplates
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/rancher/rancher/tests/v2/actions/clusters"
+	"github.com/rancher/rancher/tests/v2/actions/clustertemplates"
+	"github.com/rancher/rancher/tests/v2/actions/provisioning"
+	"github.com/rancher/rancher/tests/v2/actions/provisioninginput"
+	"github.com/rancher/shepherd/clients/rancher"
+	mgmt "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	extensionscluster "github.com/rancher/shepherd/extensions/clusters"
+	"github.com/rancher/shepherd/extensions/clusters/kubernetesversions"
+	"github.com/rancher/shepherd/extensions/settings"
+	"github.com/rancher/shepherd/pkg/config"
+	"github.com/rancher/shepherd/pkg/namegenerator"
+
+	"github.com/rancher/shepherd/pkg/session"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+var (
+	Questions = []mgmt.Question{
+		0: {
+			Variable: "rancherKubernetesEngineConfig.kubernetesVersion",
+			Default:  "",
+			Required: clustertemplates.IsRequiredQuestion,
+			Type:     "string",
+		},
+		1: {
+			Variable: "rancherKubernetesEngineConfig.network.plugin",
+			Default:  "",
+			Required: clustertemplates.IsRequiredQuestion,
+			Type:     "string",
+		},
+		2: {
+			Variable: "rancherKubernetesEngineConfig.services.etcd.backupConfig.intervalHours",
+			Default:  "",
+			Required: clustertemplates.IsRequiredQuestion,
+			Type:     "int",
+		},
+	}
+)
+
+type ClusterTemplateProvisioningTestSuite struct {
+	suite.Suite
+	client             *rancher.Client
+	session            *session.Session
+	provisioningConfig *provisioninginput.Config
+}
+
+func (ct *ClusterTemplateProvisioningTestSuite) TearDownSuite() {
+	ct.session.Cleanup()
+}
+
+func (ct *ClusterTemplateProvisioningTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	ct.session = testSession
+
+	ct.provisioningConfig = new(provisioninginput.Config)
+	config.LoadConfig(provisioninginput.ConfigurationFileKey, ct.provisioningConfig)
```

Comment:
I still feel strongly that these need to be ran as a standard user with permissions to create rke1Templates

Reply:
I agree we do have to run as a standard user for provisioning but we also have to make sure in this case for admins they do work because IMO adding additional permissions to create cluster templates is on the customer. we do not provide access to standard users directly to create rke templates with just standard user permissions